react-graph/src/PointNode.tsx

```diff
@@ -82,39 +313,91 @@ export class PointNode {
                 i > 0 && (this.points != null ? this.points[i - 1][0] : 0) <= Tend);
         if (this.points != null)
             return this.points.filter(pt => pt[0] >= Tstart && pt[0] <= Tend);
-        const result: [...number[]][] = [];
+        const result: number[][] = [];
         return result.concat(...this.children!.filter(node =>
             (node.minT <= Tstart && node.maxT > Tstart) ||
             (node.maxT >= Tend && node.minT < Tend) ||
             (node.minT >= Tstart && node.maxT <= Tend)
         ).map(node => node.GetData(Tstart, Tend, IncludeEdges)));
     }
 
-    public GetFullData(): [...number[]][] {
+    /**
+     * Retrieves all data points stored in the PointNode tree.
+     * @returns An array of all points in the tre.
+     */
+    public GetFullData(): number[][] {
         return this.GetData(this.minT, this.maxT);
     }
 
+    /**
+     * Retrieves the count of data points within a specified time range.
+     * @param Tstart Start time of the timerange to be looked at.
+     * @param Tend End time of the timerange to be looked at.
+     * @param IncludeEdges Optional parameter to include edge points in the count.
```

Comment:
technically `IncludeEdges` literally just adds 2 to the count.... so we an simplify that logic